js/index.js

```diff
@@ -0,0 +1,17 @@
+const menu = document.getElementById('mobile-menu');
+const menuList = document.querySelector('.mobile-menu-list');
+const introPage = document.querySelector('.intro-page');
+const times = document.querySelector('.fa-times');
+const listitem = document.querySelector('.mobile-menu-list');
+menu.addEventListener('click', () => {
+  introPage.style.display = 'none';
+  menuList.style.display = 'flex';
+});
+times.addEventListener('click', () => {
```

Comment:
- [ ] The mobile menu still looks the same. Please fix the issue raised by the previous reviewer.

<img src="https://user-images.githubusercontent.com/65852621/136774580-0bc03741-4a62-4568-9646-12f965543154.png" height="250px"/>